src/classes/NPSP_Contact.cls

```diff
@@ -117,16 +118,74 @@ public inherited sharing class NPSP_Contact {
         return contact.AccountId;
     }
 
+    public Id householdId() {
```

Comment:
Should this be "legacyHousholdId" or anything to differentiate it from a household account id? or is that nomenclature not used everywhere?

Reply:
I think this should stay as is because NPSP does need an IHousehold/s interface.  Then NPSP_Household and NPSP_HouseholdAccount will both have a "householdId()" method, each of which will return its respective (Account or Household__c) Id.

Reply:
@kenlewis-sfdo To @ptewson-sfdo's point though, the method name is `householdId()`. If Account represents a household in the HouseholdAccount Model, then this method should return the AccountId, but it doesn't. I think the method name needs to be different, and the method (and some others here) probably need comments documenting behavior so it's clear how to use these and when.

Reply:
I think this method on this class should keep this name for now.  It should go away entirely eventually.  https://salesforce.enterprise.slack.com/files/WUC2LP0KH/F022PRUFZR7/ihousehold.png